src/api/v1/router.ts

```diff
@@ -143,3 +150,38 @@ router.post('/service/wallet/import', async (req: Request, res: Response) => {
     return res.status(500).send({ error });
   }
 });
+
+router.post('/transaction/sign', async (req: Request, res: Response) => {
+  try {
+    const { accountId, transaction, encryptedWallet } = req.body;
```

Comment:
does the transaction include the estimated gas & nonce? I can't tell because there aren't type annotations. 
 Because the cage will not be able to look this these values - it doesn't allow outbound network requests.

Reply:
it should be string that contains serialized unsigned transaction, result of `ethers` `Transaction` method:
https://docs.ethers.org/v6/api/transaction/#Transaction-unsignedSerialized